solutions/usaco-1038.mdx

```diff
@@ -2,15 +2,20 @@
 id: usaco-1038
 source: USACO Silver 2020 US Open
 title: Social Distancing
-author: Qi Wang (Java), Tanish Tyagi (C++)
+author: Qi Wang (Java), Tanish Tyagi (C++), Melody Yu (Video)
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_socdist_silver_open20.html)
 
+## Video Solution
+
+Note: Video solution might not be the same as written solution.
+<Youtube id="mt8QhnFOkr0" />
+
 **Time Complexity:** $\mathcal{O}((N+M)\log (maxDist))$
```

Comment:
maybe move this under `## C++ Implementation`?

Reply:
The code is in C++ but the algorithm description (first half or so) applies to any language; I'd be okay with moving it if needed, though.

Reply:
oops, i meant the time complexity, cause right now it looks like the time complexity for the video instead of the code

Reply:
Oh I think that was put there by someone else before I committed https://github.com/cpinitiative/usaco-guide/blob/master/solutions/usaco-1038.mdx

Reply:
yeah, i know you didn't put that, but because the video was added it looks kind of confusing now. can you move it below the implementation header? 

or even better; move the header and tc outside of the language section (replace C++ Implementation with just Implementation)
something like this https://usaco.guide/general/adding-solution#steps

Reply:
Got it. How is it now?

Reply:
Yep, it looks good now!